docs/guides/how_to_guides/creating_batches/how_to_load_a_pandas_dataframe_as_a_batch.rst

```diff
@@ -1,18 +1,145 @@
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. content-tabs::
 
+    .. tab-container:: tab0
+        :title: Show Docs for Stable API (up to 0.12.x)
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+        .. admonition:: Prerequisites: This how-to guide assumes you have already:
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+            - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+            - Configured a :ref:`Pandas/filesystem Datasource <how_to_guides__configuring_datasources>`
+            - Identified a pandas DataFrame that you would like to use as the data to validate.
+
+        Steps
+        -----
+
+        1. Construct batch_kwargs
+
+            batch_kwargs describe the data you plan to validate. Here we are using a datasource you have configured and passing in a DataFrame under the ``"dataset"`` key.
+
+            .. code-block:: python
+
+                batch_kwargs = {
+                    "datasource": "insert_your_datasource_name_here",
+                    "dataset": insert_your_dataframe_here
+                    "data_asset_name": "optionally_insert_your_data_asset_name_here",
+                }
+
+        2. Obtain an Expectation Suite
+
+            .. code-block:: python
+
+                suite = context.get_expectation_suite(insert_your_expectation_suite_name_here)
+
+            Alternatively, if you have not already created a suite, you can do so now.
+
+            .. code-block:: python
+
+                suite = context.create_expectation_suite(insert_your_expectation_suite_name_here)
+
+        3. Get the Batch to validate
+
+            .. code-block:: python
+
+                batch = context.get_batch(
+                    batch_kwargs=batch_kwargs,
+                    expectation_suite_name=suite
+                )
+
+
+        Now that you have a :ref:`Batch <reference__core_concepts__datasources>`, you can use it to create :ref:`Expectations <expectations>` or validate the data.
+
+
+    .. tab-container:: tab1
+        :title: Show Docs for Experimental API (0.13)
+
+
+        What used to be called a “batch” in the old API was replaced with :ref:`Validator <reference__core_concepts__validation>`. A Validator knows how to validate a particular batch of data on a particular :ref:`Execution Engine <reference__core_concepts>` against a particular :ref:`Expectation Suite <reference__core_concepts__expectations__expectation_suites>`. In interactive mode, the Validator can store and update an Expectation Suite while conducting Data Discovery or Exploratory Data Analysis.
+
+        You can read more about the core classes that make GE run in our :ref:`Core Concepts reference guide <reference__core_concepts>`.
```

Comment:
Always spell out Great Expectations, see the style guide.

Reply:
Oh man I looked for this exact thing. Sorry it slipped through.